sdk/storage/azure-storage-file-datalake/src/main/java/com/azure/storage/file/datalake/DataLakePathAsyncClient.java

```diff
@@ -676,17 +676,22 @@ public DataLakeServiceVersion getServiceVersion() {
     /**
      * Package-private rename method for use by {@link DataLakeFileAsyncClient} and {@link DataLakeDirectoryAsyncClient}
      *
+     * @param destinationFileSystem The file system of the destination within the account.
```

Comment:
Any reason this comment doesn't state that `null` will use the current file system as the other comments have?

Reply:
It's internal so I didn't bother changing it, I can change it though.